### ISSUE TITLE
Live Server was supported for no workspace (just a single html file is opened). feature request

### ISSUE THREAD

**Issue by Rakesh-webdeveloper** [ritwickdey/vscode-live-server#163]
## I'm submitting a...
feature request. You have removed this feature "Live Server was supported for no workspace (just a single html file is opened).". I would be helpful if you support it again.
Usually, I write the small piece of code only for testing, in this case, I can test more faster.

```html
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  <!-- Please search GitHub for a similar issue or PR before submitting -->
[x] Feature request
[ ] Documentation issue or request
[ ] Other: <!-- Please describe: -->
```

## Environment

```html
Browser:

- [ ] Chrome (desktop) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] IE version XX
- [ ] Edge version XX

For Tooling issues:

- Live Server: <!-- Check which version is installed --> 
- Platform:  <!-- Mac, Linux, Windows -->
- Visual Studio Code: <!-- `code --version` -->
```

**Comment by imsteaky** [comment]
is that where my error "Open a folder or workspace... (File -> Open Folder)" is coming from when I try to open up liveserver for.. anything?

**Comment by ritwickdey** [comment]
Yep! It is dropped to support multi-root workspace. If we now enable this feature anyhow, the extension will crash unpredictably. We need to handle the edge cases. 

If anyone likes to submit a PR, then it'll be nice - just have to handle edge cases by if-else-if-else block. 🙍

**Comment by imsteaky** [comment]
Thanks for the clarification! I understand the change now and it wasn't as big of a deal to adjust my workflow as I initially thought.  Was just very confused that first day why live server wasn't opening individual files properly. 

Liveserver is amazing and I can't imagine not having it. Thanks!

**Comment by kashifali554** [comment]
I am a new user for live server and want to take this extension for a spin but unable to do so because of this error.
I am using MacOS 10.14.1
VS Code: 1.29.0
Live Server: 5.1.1

How can I fix this issue while opening a single file. ie, index.html?
